modules/RTC/TraceablePeerConnection.js

```diff
@@ -2068,6 +2143,10 @@ TraceablePeerConnection.prototype.setMaxBitRate = function(localTrack = null) {
 TraceablePeerConnection.prototype.setRemoteDescription = function(description) {
     this.trace('setRemoteDescription::preTransform', dumpSDP(description));
 
+    // Munge the order of the codecs based on the preferences set through config.js
+    // eslint-disable-next-line no-param-reassign
+    description = this._mungeCodecOrder(description);
```

Comment:
I'm not sure what are those reassigns doing all over the `setRemoteDescription`. By no means it wasn't meant as a guideline. It probably creeped in when eslint was enabled for the first time and the intention was to do least changes possible, so ignore were added everywhere. But you can keep this one since it's all over already, but not something to do in the future.